packages/dds/tree/src/simple-tree/objectNode.ts

```diff
@@ -47,7 +47,7 @@ import { TreeNodeValid, type MostDerivedData } from "./treeNodeValid.js";
 import { getUnhydratedContext } from "./createContext.js";
 
 /**
- * Helper used to produce types for object nodes.
+ * Generates the properties for an ObjectNode from its field schema object.
```

Comment:
I'm not sure I like "generates" for a type, which has no runtime effect, even if in some sense it does generate something at compile time. Thoughts on "Type which has the same properties as an ObjectNode created from the specified field schema object"?